### ISSUE TITLE
"\n" breaks line in command usage help message instead of being displayed

### ISSUE THREAD

**Issue by PovilasLT** [Zrips/CMI#8640]
### Description of Issue

When sending `/titlemsg` without any args, help message renders `\n` as new line instead of showing it in chat.

![image](https://github.com/Zrips/CMI/assets/16564979/14e8c80c-5ac5-48af-a668-dc89b3ac6d8d)
![image](https://github.com/Zrips/CMI/assets/16564979/6286bd85-fbfa-46ab-86fb-daed2084347d)

### Version Information

```shell
[00:03:43 INFO]: CMI: 9.7.0.4 Unknown  MySQL-> 9.7.0.6
[00:03:43 INFO]: CMILib: 1.4.6.2
[00:03:43 INFO]: Server: Paper(477) 1.20.4-R0.1-SNAPSHOT-
[00:03:43 INFO]: CMI economy: Įjungtas Vault: 1.7.3-b CMI Chat: Įjungtas
[00:03:43 INFO]: Modules -> 52 enabled 7 disabled: hpBossBar, chatBubble, tablist, ranks, firstJoinMessages, playerCombat, votifier
```

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by mrfloris** [comment]
Yes, that's correct. The `\n` will help you define where the title ends and subtitle starts

`/cmi titlemsg all meow \n says the cat`

this will show a big title meow, with sub-title cat, the backslash n will get parsed - and not shown as plaintext. This is on purpose.

**Comment by Zrips** [comment]
Should be resolved with next CMI and CMILib updates
